development_environment

```diff
@@ -117,6 +117,15 @@ OUTPUT_ROOT_DIR=/docker-entrypoint-initdb.d
 
 FLOWKIT_INTEGRATION_TESTS_DISABLE_AUTOSTART_SERVERS=FALSE
 
-# Worked examples
+# FlowETL
+DAGS_FOLDER=/usr/local/airflow/src
+SQL_ALCHEMY_CONN=postgres://flowetl:flowetl@flowetl_db:5432/flowetl
+FERNET_KEY=ssgBqImdmQamCrM9jNhxI_IXSzvyVIfqvyzES67qqVU=
```

Comment:
I don’t think it matters, but won’t this clash with the env var for flowauth?

Reply:
Might be an issue if having the same key for both opens us up to some kind of attack? Will add prefix to them...